Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml

```
@@ -283,6 +283,20 @@
   - type: AccessReader
     access: [["Armory"]]
 
+- type: entity
+  parent: CrateBaseSecure
+  suffix: Armory, Secure
+  id: CrateContrabandStorageSecure
+  name: contraband storage crate
+  description: An armory access locked crate for storing contraband confiscated from suspects or prisoners.
+  components:
+  - type: Icon
+    sprite: Structures/Storage/Crates/sec_gear.rsi
+  - type: Sprite
```

This icon looks unnecessary so I'ma delete it.

```suggestion
  - type: Sprite
```